CONTRIBUTING.md

```diff
@@ -41,7 +41,7 @@ Docker allows us to "containerize" and run our application as if it were being h
 	npm run dev
 	```
 
-	This will host the website on `localhost`, which can keep running while you are editing the project; any browser tabs will "hot reload" with your changes.
+	This will host the website on `localhost`, which you can open in a local browser: see the address ouput by the final command above. You can keep this site running while you are editing the project; any browser tabs will "hot reload" with your changes when you save them.
```

Comment:
```suggestion
	This will host the website on `localhost`, which you can open in a local browser: see the address output by the final command above. You can keep this site running while you are editing the project; any browser tabs will "hot reload" with your changes when you save them.
```